data/core/common.lua

```diff
@@ -869,4 +869,69 @@ function common.rm(path, recursively)
 end
 
 
+---Open the given resource using the system launcher.
+---The resource can be an url, file or directory in most cases...
+---@param resource string
+---@return boolean success
+function common.open_in_system(resource)
+  -- Detect platforms
+  local function detect_platform()
+    if PLATFORM:lower():find("unknown", 1, true) then
+      -- AmigaOS: ENV: is a system assign
+      local f = io.open("ENV:", "r")
+      if f then f:close() return "amiga" end
+      -- Try uname
+      local uname = process.start({"uname", "-s"})
+      if uname then
+        uname:wait(1000)
+        return uname:read_stdout(128):lower()
+      end
+      return "unknown"
+    end
+    return PLATFORM:lower()
+  end
+
+  local function is_url(str)
+    return str:match("^[a-z]+://")
+  end
+
+  local launcher
+  local platform = detect_platform()
+
+  if platform:find("windows", 1, true) then
+    launcher = 'start ""'
+  elseif
+    platform:find("mac", 1, true)
+    or platform:find("haiku", 1, true)
+    or platform:find("serenity", 1, true)
+  then
+    launcher = "open"
+  elseif
+    platform:find("amiga", 1, true)
+    or platform:find("morph", 1, true)
+    or platform:find("aros", 1, true)
```

Comment:
This doesn't need to be supported. Currently it doesn't have a port of SDL2, nor SDL3